styles/globals.css

```diff
@@ -1,3 +1,18 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer utilities {
```

Comment:
does this mean the users have to implement the following code as well?

Reply:
good catch, fixed in 4386d62e5b55a14baa670cab09175d64ef78f318